src/OpenTok/OpenTok.php

```diff
@@ -215,9 +215,29 @@ public function createSession($options=array())
      * set to routed); you cannot archive sessions with the media mode set to relayed.
      *
      * @param String $sessionId The session ID of the OpenTok session to archive.
-     * @param String $name The name of the archive. You can use this name to identify the archive.
-     * It is a property of the Archive object, and it is a property of archive-related events in the
-     * OpenTok JavaScript SDK.
+     *
+     * @param array $options A set of options for the archive. This array includes the
```

Comment:
we need to note the deprecated method signature as well. is there a way yo use PHPDoc to say the second param is an array or a string, and that the string is deprecated?